R/travis-auth.R

```diff
@@ -1,75 +1,103 @@
-auth_travis_ <- function(gh_token = NULL) {
+#' Authenticate to Travis
+#' @description
+#'   Authenticates to Travis and asks to create an API key if none exists.
+#' @template endpoint
+#' @export
+#'
+auth_travis <- function(endpoint = ".org") {
+  yml <- tryCatch(
+    {
+      readLines("~/.travis/config.yml")
```

Comment:
Can we `stop()` if the file doesn't exist, with instructions to set up such a file?

Reply:
Currently we create the file ourselves if it does not exist in `wait_for_clipboard_token()`. Is this bad?

I've added some more cli output so that the user gets informed what is happening:

```r
Writing Travis CI API token for endpoint '.org' to ~/.travis/config.yml.
```